securedrop/tests/functional/functional_test.py

```diff
@@ -31,19 +30,31 @@ class FunctionalTest():
 
     def _unused_port(self):
         s = socket.socket()
-        s.bind(("localhost", 0))
+        s.bind(('127.0.0.1', 0))
         port = s.getsockname()[1]
         s.close()
         return port
 
     def _create_webdriver(self):
-        log_file = open('tests/log/firefox.log', 'a')
-        log_file.write(
-            '\n\n[%s] Running Functional Tests\n' % str(
-                datetime.now()))
+        log_file = open('tests/log/tbb.log', 'a')
+        log_msg = '\n\n[%s] Running Functional Tests\n' % str(datetime.now())
+        log_file.write(log_msg)
         log_file.flush()
-        firefox = firefox_binary.FirefoxBinary(log_file=log_file)
```

Comment:
You should preserve use of a FF log file, to aid in debugging TBB/FF issues in the future.